src/selectors/prometheus/selectors.js

```diff
@@ -28,3 +28,40 @@ export const getLastUtilizationStat = response => {
   const history = getUtilizationVectorStats(response);
   return history ? history[history.length - 1] : null;
 };
+
+export const getTopConsumerVectorStats = result => {
+  let maxVal = 0;
+
+  result.forEach(namespace => {
```

Comment:
you can use map here instead

Reply:
I don't think map can be used here. Map method creates a new array with the results of calling a provided function on every element in the calling array. Here, I don't want to create a new array after performing the required actions. I just want to iterate over the array in order to find the max element. Hence, used forEach.